src/components/cms/SkillCreator/SkillWizard.js

```diff
@@ -717,9 +717,25 @@ class SkillWizard extends Component {
       });
       return 0;
     }
-    if (category === null || language === '' || name === '') {
+    if (name === '') {
       this.props.actions.openSnackBar({
-        snackBarMessage: 'Please select a group, language and a skill',
+        snackBarMessage: 'Please give a skill name',
+        snackBarPosition: { vertical: 'top', horizontal: 'right' },
+        variant: 'warning',
+      });
+      return 0;
```

Comment:
It involves a lot of redundant code, instead write a function that returns the missing field.